pages/common/lwp-request.md

```diff
@@ -0,0 +1,27 @@
+# lwp-request
+
+> Simple Perl-based command-line HTTP client.
```

Comment:
It's certainly informative, and worth having IMO, to make it clear what lwp means.